application.lisp

```diff
@@ -1,6 +1,16 @@
 (in-package :aark)
 
-(defparameter +delay+ (/ 1000.0 30.0))	; fps
+(defparameter +delay+ (/ 1000.0 30.0)
+  "Frames per second.")
+
+(defparameter +root+ (asdf:system-source-directory :aark)
+  "Absolute project root path.")
+
+;; TODO: provide something like config package with resource path
+;;       mappings and path-related routines. Now I can only hardcode them.
+(defmacro -res- (resource-file-name)
```

Comment:
I think we can ditch dashes in `-res-`, since it's not conventional syntax, unlike `*var*` for global variables or `+const+` for global constants.

Reply:
Ok, its just tempting to use some names I can't in other languages, you know.